app/components/document_body_component.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= render Arclight::MetadataSectionComponent.with_collection(
+      component_metadata_partials,
+      metadata_attr: { layout: Arclight::UpperMetadataLayoutComponent },
+      presenter: presenter
+    ) %>
+<%= embed_html %>
```

Comment:
The embed should render on the collection page too

Reply:
Ah, I didn't realize that. I can put it back in the parent.